Source/Core/Core/PatchEngine.cpp

```diff
@@ -191,15 +196,37 @@ static void ApplyPatches(const std::vector<Patch>& patches)
       {
         u32 addr = entry.address;
         u32 value = entry.value;
+        u32 comparand = entry.comparand;
         switch (entry.type)
         {
         case PatchType::Patch8Bit:
+          if (entry.conditional)
+          {
+            if (PowerPC::HostRead_U8(addr) != static_cast<u8>(comparand))
+            {
+              break;
+            }
+          }
           PowerPC::HostWrite_U8(static_cast<u8>(value), addr);
```

Comment:
My personal preference would be to write this like:

```suggestion
          if (!entry.conditional || PowerPC::HostRead_U8(addr) == static_cast<u8>(comparand))
            PowerPC::HostWrite_U8(static_cast<u8>(value), addr);
```

But I'm okay with the current implementation if others prefer that.

Reply:
It seems unnecessarily verbose to me right now. At the very least, the two conditions should be joined into a single `if` (unless there are plans to have other conditionals in the future.)
I'm impartial to leaving the `break` vs. inverting the condition for the write.

Reply:
I opted for JosJuice's suggestion.